src/Service.Host/client/src/components/parts/mechPartSource/tabs/ProposalTab.js

```diff
@@ -104,14 +119,23 @@ function ProposalTab({
                     <Dropdown
                         label="Part Type*"
                         propertyName="partType"
-                        items={partTypes}
+                        items={electricalPartTypes}
                         fullWidth
                         value={partType}
                         onChange={handleFieldChange}
                     />
                 </Grid>
             ) : (
-                <Grid item xs={3} />
+                <Grid item xs={3}>
+                    <Dropdown
+                        label="Part Type*"
+                        propertyName="partType"
+                        items={mechanicalPartTypes}
+                        fullWidth
+                        value={partType}
+                        onChange={handleFieldChange}
+                    />
```

Comment:
would probably be better to just to vary  the items list according to partType === 'E' as opposed to whole Dropdown component